src/modules/ims_icscf/ims_icscf_mod.c

```diff
@@ -71,7 +71,7 @@ stat_var *uar_replies_response_time = NULL;
 stat_var *lir_replies_response_time = NULL;
 
 //module parameters
-char *ims_icscf_db_url = "mysql://icscf:heslo@localhost/hssdata"; /**< DB URL */
+char *ims_icscf_db_url = "mysql://kamailio:kamailiorw@localhost/kamailio"; /**< DB URL */
```

Comment:
Consider using the correct define for it: DEFAULT_RODB_URL

Reply:
Thanks, I resolved it.